### DISCUSSION TITLE
Gradle build fails on Mac OS

### DISCUSSION THREAD

**Discussion by violetagg** [community/community#26512] [Actions]
<p>Hi,</p>
<p>I have a workflow that tests the build on different OSes.</p>
<p><a href="https://github.com/reactor/reactor-netty/actions/runs/57443581" rel="nofollow noopener">https://github.com/reactor/reactor-netty/actions/runs/57443581</a></p>
<p>The build is not OK on MAC OS and fails with the log below (the build does not fail on the other OSes)</p>
<pre><code>&gt; Could not resolve all dependencies for configuration ':compile'.
   &gt; Could not resolve io.netty:netty-handler-proxy:4.1.48.Final-SNAPSHOT.
     Required by:
         project :
      &gt; Could not resolve io.netty:netty-handler-proxy:4.1.48.Final-SNAPSHOT.
         &gt; Unable to load Maven meta-data from https://oss.sonatype.org/content/repositories/snapshots/io/netty/netty-handler-proxy/4.1.48.Final-SNAPSHOT/maven-metadata.xml.
            &gt; Could not get resource 'https://oss.sonatype.org/content/repositories/snapshots/io/netty/netty-handler-proxy/4.1.48.Final-SNAPSHOT/maven-metadata.xml'.
               &gt; Could not GET 'https://oss.sonatype.org/content/repositories/snapshots/io/netty/netty-handler-proxy/4.1.48.Final-SNAPSHOT/maven-metadata.xml'. Received status code 403 from server: Forbidden
      &gt; Could not resolve io.netty:netty-handler-proxy:4.1.48.Final-SNAPSHOT.
         &gt; Unable to load Maven meta-data from https://oss.sonatype.org/content/repositories/releases/io/netty/netty-handler-proxy/4.1.48.Final-SNAPSHOT/maven-metadata.xml.
            &gt; Could not get resource 'https://oss.sonatype.org/content/repositories/releases/io/netty/netty-handler-proxy/4.1.48.Final-SNAPSHOT/maven-metadata.xml'.
               &gt; Could not GET 'https://oss.sonatype.org/content/repositories/releases/io/netty/netty-handler-proxy/4.1.48.Final-SNAPSHOT/maven-metadata.xml'. Received status code 403 from server: Forbidden
</code></pre>
<p>What could be the problem?</p>
<p>Thanks,<br>
Violeta</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/violetagg">@violetagg</a> ,</p>
<p>This is due to macOS hosted runner ip is blocked by Maven center, and now they have re-enabled the ip.</p>
<p>I forked your repo and the issue is gone on my side.  Please check your workflow for confirmation.</p>
<p>Here is my <a href="https://github.community/t/gradle-fetching-library-from-oss-sontatype-org-throws-403-although-library-is-public/17857/8">answer</a> in similar ticket.</p>
<p>Thanks.</p>

**Comment by violetagg** [comment]
<p>It is working now</p>
<p>Thanks</p>

**Comment by claudani** [comment]
<p>Hi, I seem to be having the exact same problem. Is there a way I could check if the Mac’s IP is blocked and/or unblock it myself? Also I’m quite new to the Gradle world, how does one “enable Gradle caching on their workflow”? Thanks in advance for the help! </p>
